File: examples/ajax/twitter.js
user = 'gkossakowski'
numTweets = 5
appendTo = "#jstwitter"

function loadTweets() {
    $.ajax({
        url: 'http://api.twitter.com/1/statuses/user_timeline.json/',
        type: 'GET',
        dataType: 'jsonp',
        data: {
            screen_name: user,
            include_rts: true,
            count: numTweets,
            include_entities: true
        },
        success: function(data, textStatus, xhr) {
            for (var i=0; i < data.length; i++) {
                $(appendTo).append(
                    '<p>' + data[i].text + '</p>'
                )
            }
        }   
    })
}

$(document).ready(loadTweets)

File: examples/ajax/twitter.scala
val numTweets = 5
val appendTo = "#jstwitter"

for (user <- array("foo", "bar", "baz")) {
  log("fetching " + user)
  val data = ajax.get(
    new JSLiteral {
      val url = "http://api.twitter.com/1/statuses/user_timeline.json",
      val `type` = "GET",
      val dataType = "jsonp",
      val data = new JSLiteral {
        val screen_name = user,
        /* ... */
      }
    }
  )

  val user_div = "user_" + user
  $(appendTo).append("<div id='" + user_div + "'>")
  for (d <- data) {
    $("#" + user_div).append("<p>" + d.text + "</p>")
  }
}
